GitUI/UserControls/RevisionGrid/RevisionDataGridView.BackgroundUpdater.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
```

Comment:
No need for `ArgumentNullException` in ctor with
```suggestion
#nullable enable

﻿using System;
```

Reply:
To be clear, the nullable feature does not generate any runtime checking of values.

Reply:
Also if this is rebased on top of my PR, then nullable is enabled for all files by default in most projects. I have another PR ready to enable it for some of those that remain.